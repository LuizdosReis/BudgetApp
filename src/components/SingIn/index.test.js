import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { SignInPage } from './index';

let wrapper;
let startLogin;

beforeEach(() => {
  startLogin = jest.fn();
  wrapper = shallow(<SignInPage startLogin={startLogin} />);
});

test('should render SingIn correctly', () => {
  expect(toJson(wrapper)).toMatchSnapshot();
});

test('should handle onClick', () => {
  wrapper.find('button').simulate('click');
  expect(startLogin).toBeCalled();
});
