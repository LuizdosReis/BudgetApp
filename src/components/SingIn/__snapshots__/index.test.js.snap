// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SingIn correctly 1`] = `
<main
  className="content"
>
  <button
    className="pure-button"
    onClick={[Function]}
    type="submit"
  >
    Entrar
  </button>
</main>
`;
